mteb/evaluation/evaluators/Image/Any2AnyRetrievalEvaluator.py

```diff
@@ -36,7 +36,7 @@
 
 logger = logging.getLogger(__name__)
 
-transform = transforms.Compose([transforms.PILToTensor()])
+DEFAULT_TRANSFORM = transforms.Compose([transforms.PILToTensor()])
```

Comment:
> Shouldn't it be up to the model to decide how they want to transform their images?

this was more of a default transform to enable the image Dataset to work with Dataloader&encoding. The change makes it more flexible though! If I remember correctly, if transform is None, it will run into errors for datasets with images of different resolutions?

Reply:
> > Shouldn't it be up to the model to decide how they want to transform their images?
> 
> this was more of a default transform to enable the image Dataset to work with Dataloader&encoding. The change makes it more flexible though! If I remember correctly, if transform is None, it will run into errors for datasets with images of different resolutions?

Yes, therefore I have maintained this default transform, and existing models won't be affected by this code change.